### ISSUE TITLE
Fix deprecation warnings 

### ISSUE THREAD

**Issue by wangruohui** [open-mmlab/mmediting#733]
Some warnings may affect functionality, we need to fix them.

```
tests/test_data/test_datasets/test_sr_dataset.py::test_sr_test_multiple_gt_dataset
  /home/runner/work/mmediting/mmediting/mmedit/datasets/sr_test_multiple_gt_dataset.py:32: UserWarning: "SRTestMultipleGTDataset" have been deprecated and will be removed in future release. Please use "SRFolderMultipleGTDataset" instead. Details see https://github.com/open-mmlab/mmediting/pull/355
    warnings.warn('"SRTestMultipleGTDataset" have been deprecated and '
```

```
tests/test_models/test_restorers/test_real_basicvsr.py::test_real_basicvsr
  /opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/torch/nn/functional.py:3658: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details. 
    "The default behavior for interpolate/upsample with float scale_factor changed "
```

```
tests/test_models/test_backbones/test_sr_backbones/test_basicvsr_plusplus.py::test_basicvsr_plusplus
  /opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/torch/nn/functional.py:2973: UserWarning: Default upsampling behavior when mode=bicubic is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
    "See the documentation of nn.Upsample for details.".format(mode))
```

**Comment by wangruohui** [comment]
Other fixes: https://github.com/open-mmlab/mmediting/pull/728

**Comment by wangruohui** [comment]
Fixed in #731 
```
/home/runner/work/mmediting/mmediting/tests/test_data/test_datasets/test_vfi_dataset.py:7: PytestCollectionWarning: cannot collect test class 'TestVFIDataset' because it has a __init__ constructor (from: tests/test_data/test_datasets/test_vfi_dataset.py)
    class TestVFIDataset:
```
